Examples/BookingSystem.AspNetCore/Stores/SessionStore.cs

```diff
@@ -385,7 +385,13 @@ protected override void BookOrderItems(List<OrderItemContext<SessionOpportunity>
                     RenderOpportunityId(ctxGroup.Key).ToString(),
                     RenderOfferId(ctxGroup.Key).ToString(),
                     ctxGroup.Count(),
-                    false);
+                    false,
+                    ctxGroup.ToList().Select(ctx => new OpenActive.FakeDatabase.NET.FakeDatabase.RequestBarCode
+                    {
+                        Url = ctx.RequestOrderItem.AccessPass?.FirstOrDefault()?.Url.ToString(),
```

Comment:
Note to self: providing a URL doesn't make sense in this context as it won't be read by the booking system. Only `test` can be provided`. Clarify the spec accordingly.